services/ui-src/src/components/fields/DropdownField.tsx

```diff
@@ -152,5 +158,12 @@ interface Props {
   nested?: boolean;
   autosave?: boolean;
   sxOverride?: AnyObject;
+  styleAsOptional?: boolean;
   [key: string]: any;
 }
+
+const sx = {
+  ".optional-text": {
+    fontWeight: "lighter",
+  },
+};
```

Comment:
Request: As noted above, this style should be moved to `Form.tsx` and only be written there once. Then we can remove the `sx` object here. Then you can revert the change on line 134 back to `sx={sxOverride}`.